docs/code_ref/coordinates/rotatedsunframe.rst

```diff
@@ -0,0 +1,390 @@
+.. _sunpy-coordinates-rotatedsunframe:
+
+Differential rotation using coordinate frames
```

Comment:
I think in this section there should be a clear reference to the formula (or paper that contains it) that is used for the differential rotaiton